### DISCUSSION TITLE
[Allure TestOps]  Can't remove/rename default layer name

### DISCUSSION THREAD

**Discussion by bratashX** [allure-framework/allure2#1962] [TestOps]
Can't remove the default Allure TestOps layers:

- API Tests
- UI Tests
- Unit Tests

![image](https://user-images.githubusercontent.com/68950226/229083272-7173c435-7194-48b2-b2f7-269d889fc6dd.png)

**Comment by cheshi-mantu** [comment]
@bratashX 
that's all correct. You cannot do it.
